Improved the bulk operation actions description text.

navneet0693 opened the pull request:
## Problem
The text "Select / unselect all @count results in this view'" and "Clear selected Members" in Manage members tab in groups are a bit misleading and can be improved for better reading.

## Solution
//s "Select / unselect all @count results in this view'" to "Select / unselect all @count members across all the pages"
//s "Clear all selected members" to "Clear selection"

## Issue tracker
https://www.drupal.org/project/social/issues/3094403

## How to test
- [ ] Go to manage members tab in a group
- [ ] Verify that the checkbox for selecting all members text "Select / unselect all  {count} members across all the pages"
- [ ] On the same page verify that the button for clear the selections have "Clear selection" label.
- [ ] Enable social_event_managers module.
- [ ]  Go Manage Enrollments tab in an event
- [ ] Verify that the checkbox for selecting all members text "Select / unselect all  {count} members across all the pages"
- [ ] On the same page verify that the button for clear the selections have "Clear selection" label.

## Release notes
N.A

## Change Record
N.A

xinyumaGG commented:
Hi @navneet0693 

Maybe we can make the clear selection more informative, does it: "Clear all selected members" or "Clear all selected members on this page"?

navneet0693 commented:
@xinyumaGG "Clear all selected member on this page" may also mean (or can be confused) for that it will to "delete" or "remove" from group or event while "clear selection" means reset all selection.

xinyumaGG commented:
you are right, maybe we should say something like "clear selection on all pages"? 

We just need to further clarify on which page the selections will be cleared :-D

navneet0693 commented:
> The change in string looks good :)
> 
> However, we shouldn't manually change the *.po and *.pot files. These are created for us in automated processes that'll also ensure there are no merge conflicts. So please undo those changes.
> 
> In #1600 we've added a file called `translations.php` to the project root. The PR template in #1605 has been updated to point to this. Please copy the old versions of the strings you've changed into that file as explained in the file. This ensures that projects running 6.x or 7.x versions before this change is made are still able to receive the old translated string :)

Thank you @Kingdutch I have updated the PR.

Kingdutch commented on translations.php:
```diff
@@ -20,3 +21,6 @@
 // new PluralTranslatableMarkup($count, '1 example', '@count examples');.
+// Changed in version 7.2.
+new TranslatableMarkup('Select / unselect all @count members across all the pages');
+new TranslatableMarkup('Clear selection on all pages');
```
```suggestion
new TranslatableMarkup('Clear all selected members');
```

Kingdutch commented on translations.php:
```diff
@@ -20,3 +21,6 @@
 // Changed in version X.Y
 // new TranslatableMarkup('Example');
 // new PluralTranslatableMarkup($count, '1 example', '@count examples');.
+// Changed in version 7.2.
```
Lets preserve the empty line here for readability. Coding standards tell us not to, but this file is there for documentation so we don't need coding standards. :)
```suggestion

// Changed in version 7.2.
```

navneet0693 commented:
@Kingdutch Thank you very much for review, explanation of PR build failing is given here:
https://github.com/goalgorilla/open_social/pull/1606#issuecomment-554763424 which can be seen at https://travis-ci.org/goalgorilla/open_social/jobs/613563752#L502

Kingdutch commented:
Cherry-picked on 7.x in 55931fd55
